### PR TITLE
feat: v8 python code snippets

### PR DESCRIPTION
**Pull request by Justintime50**
- Adds the new v8 code snippets
- Copied the previous current snippets to the v7 dir

### CODE REVIEW

File: official/docs/python/current/api-keys/retrieve.py [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
 # Retrieve all API keys (authenticated user and child user keys)
-api_keys = easypost.User.all_api_keys()
+api_keys = client.user.all_api_keys()
 
 # Retrieve API keys for a child user
-child_user = easypost.User.retrieve("user_...")
+child_user = client.user.retrieve("user_...")
 api_keys = child_user.api_keys()
```
Comment (jchen293) [inline official/docs/python/current/api-keys/retrieve.py:13]: Doesn't have instance method anymore, this need to be updated

File: official/docs/python/current/batches/add-shipments.py [hunk 2]
```diff
@@ -3,15 +3,16 @@
...
 
-easypost.api_key = os.getenv("EASYPOST_API_KEY")
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
-batch = easypost.Batch.retrieve("batch_...")
+batch = client.batch.retrieve("batch_...")
 
-batch.add_shipments(
+batch_with_shipments = client.batch.add_shipments(
+    client.id,
```
Comment (jchen293) [inline official/docs/python/current/batches/add-shipments.py:11]:
```suggestion
    batch.id,
```

File: official/docs/python/current/brand/update.py [hunk 3]
```diff
@@ -3,10 +3,10 @@
...
 
 
-easypost.api_key = os.getenv("EASYPOST_API_KEY")
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
-user = easypost.User.retrieve_me()
+user = client.user.retrieve_me()
 
-user = user.update_brand(color="#303F9F")
+updated_brand = client.user.update_brand(color="#303F9F")
```
Comment (jchen293) [inline official/docs/python/current/brand/update.py:10]: need pass in `id` in the first parameter

File: official/docs/python/current/carrier-accounts/update.py [hunk 4]
```diff
@@ -3,12 +3,13 @@
...
-easypost.api_key = os.getenv("EASYPOST_API_KEY")
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
-carrier_account = easypost.CarrierAccount.retrieve("ca_...")
+carrier_account = client.carrier_account.retrieve("ca_...")
 
-carrier_account.description = "FL Location DHL eCommerce Solutions Account"
-carrier_account.credentials["pickup_id"] = "abc123"
-carrier_account.save()
+updated_carrier_account = client.carrier_account.update(
```
Comment (jchen293) [inline official/docs/python/current/carrier-accounts/update.py:10]: need pass in `id` in the first parameter

File: official/docs/python/current/endshipper/update.py [hunk 5]
```diff
@@ -3,21 +3,21 @@
...
-endshipper.company = "BAZ"
-endshipper.street1 = "164 TOWNSEND STREET UNIT 1"
-endshipper.street2 = "UNIT 1"
-endshipper.city = "SAN FRANCISCO"
-endshipper.state = "CA"
-endshipper.zip = "94107"
-endshipper.country = "US"
-endshipper.phone = "555-555-5555"
-endshipper.email = "FOO@EXAMPLE.COM"
+updated_end_shipper = client.end_shipper.update(
```
Comment (jchen293) [inline official/docs/python/current/endshipper/update.py:10]: need pass in `id` in the first parameter

**Review by nwithan8 — APPROVED** [review]

File: official/docs/python/current/referral-customers/update.py [hunk 6]
```diff
@@ -3,9 +3,9 @@
 import easypost
 
 
-easypost.api_key = os.getenv("EASYPOST_API_KEY")
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
-easypost.Referral.update_email(
+client.referral_customer.update_email(
```
Comment (jchen293) [inline official/docs/python/current/referral-customers/update.py:8]: need `id` in the first parameter

File: official/docs/python/current/smartrate/retrieve-estimated-delivery-date.py [hunk 7]
```diff
@@ -3,9 +3,9 @@
 import easypost
 
 
-easypost.api_key = os.getenv("EASYPOST_API_KEY")
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
-shipment = easypost.Shipment.retrieve("shp_...")
+shipment = client.shipment.retrieve("shp_...")
 
 estimated_delivery_dates = shipment.retrieve_estimated_delivery_date(planned_ship_date="YYYY-MM-DD")
```
Comment (jchen293) [inline official/docs/python/current/smartrate/retrieve-estimated-delivery-date.py:10]: this need to be updated with client

File: official/docs/python/current/users/update.py [hunk 8]
```diff
@@ -3,12 +3,10 @@
...
 
 
-easypost.api_key = os.getenv("EASYPOST_API_KEY")
+client = easypost.EasyPostClient(os.getenv("EASYPOST_API_KEY"))
 
-user = easypost.User.retrieve_me()
+user = client.user.retrieve_me()
 
-user.recharge_threshold = "50.00"
+updated_user = client.user.update(recharge_threshold="50.00")
```
Comment (jchen293) [inline official/docs/python/current/users/update.py:10]: need `id` in the first parameter

**Review by jchen293 — CHANGES_REQUESTED** [review]

**Review by jchen293 — APPROVED** [review]
approving it now because i will be gone most of the days this week

**Pull request merged**